src/VSCodeExtension/src/commands.ts

```diff
@@ -22,13 +22,76 @@ export function registerCommand(context: vscode.ExtensionContext, name: string,
     )
 }
 
+function createNewProjectAtUri(dotNetSdk: DotnetInfo, projectType: string, uri: vscode.Uri) {
+    let proc = cp.spawn(
+        dotNetSdk.path,
+        ["new", projectType, "-lang", "Q#", "-o", uri.fsPath]
+    );
+    
+    let errorMessage = "";
+    proc.stderr.on('data', data => {
+        errorMessage = errorMessage + data;
+    });
+
+    proc.on(
+        'exit', (code, signal) => {
+            if (code === 0) {
+                const openItem = "Open new project...";
+                vscode.window.showInformationMessage(
+                    `Successfully created new project at ${uri.fsPath}.`,
+                    openItem
+                ).then(
+                    (item) => {
+                        if (item === openItem) {
+                            vscode.commands.executeCommand(
+                                "vscode.openFolder",
+                                uri
+                            ).then(
+                                (value) => {},
+                                (value) => {
+                                    vscode.window.showErrorMessage("Could not open new project");
+                                }
+                            );
+                        }
+                    }
+                );
+            } else {
+                // Check if the problem was that the project templates are missing.
+                // If so, we can give a more helpful error message and offer the
+                // user to install templates.
+                if (errorMessage.includes("'Q#' is not a valid value for -lang")) {
+                    const installTemplatesItem = "Install project templates and retry";
+                    vscode.window.showErrorMessage(
+                        "Project creation failed, as the Q# project templates not installed.",
```

Comment:
I would suggest the edit, but it doesn't let me. How about instead: 
"Project creation failed. The Q# project templates may not be installed.",

Reply:
edited.